.github/workflows/cross_platform_ci.yml

```
@@ -0,0 +1,102 @@
+name: rviz cross-platform RoboStack build
```

Shorter display name in github:
```suggestion
name: RoboStack
```